config/notifiers.go

```diff
@@ -392,3 +403,54 @@ func (c *PushoverConfig) UnmarshalYAML(unmarshal func(interface{}) error) error
 	}
 	return checkOverflow(c.XXX, "pushover config")
 }
+
+type JiraConfig struct {
+	NotifierConfig `yaml:",inline" json:",inline"`
+
+	Domain   string
+	User     string
+	Password Secret
+
+	Project     string                 `yaml:"project" json:"project"`
+	IssueType   string                 `yaml:"issueType" json:"issueType"`
+	Priority    string                 `yaml:"priority" json:"priority"`
+	Summary     string                 `yaml:"summary" json:"summary"`
+	Description string                 `yaml:"description" json:"description"`
+	Fields      map[string]interface{} `yaml:"fields" json:"fields"`
+
+	// Catches all undefined fields and must be empty after parsing.
+	XXX map[string]interface{} `yaml:",inline" json:"-"`
```

Comment:
no, the `inline` is an option, not the field name. see other such structs in this file.